atlasdb-cassandra-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CQLKeyValueServiceSweeperTest.java

```diff
@@ -28,7 +28,7 @@ protected KeyValueService getKeyValueService() {
         return CQLKeyValueService.create(
                 CassandraKeyValueServiceConfigManager.createSimpleManager(
```

Comment:
Does it make sense to extract this `CassandraKeyValueServiceConfigManager`, if multiple tests are using the same configuration?